backend/services/emailTemplates/conversationNotification/html.ejs

```diff
@@ -64,7 +64,7 @@
                               <div style="Margin-right: 10px; Margin-left: 10px;">
                                  <div style="line-height: 10px; font-size: 1px">&nbsp;</div>
                                  <div style="font-size:12px;line-height:14px;font-family:Roboto, Tahoma, Verdana, Segoe, sans-serif;color:#555555;text-align:left;">
-                                    <p style="margin: 0;font-size: 12px;line-height: 14px;text-align: center"><span style="font-size: 24px; line-height: 28px;"> <%= authorName %> , <%= text1 %> <%= userName %> <%= text2 %>!</span>
+                                    <p style="margin: 0;font-size: 12px;line-height: 14px;text-align: center"><span style="font-size: 24px; line-height: 28px;"> <%= authorName %> <%= text1 %> <%= userName %> <%= text2 %>!</span>
```

Comment:
Let's make the text in the function and pass to the template only one property instead of "authorName", "text1", "userName" and "text2"